### ISSUE TITLE
ʿAdwā, not ʿĀdwā

### ISSUE THREAD

**Issue by abausi** [BetaMasaheft/Documentation#2188]
Iit should be ʿAdwā, not ʿĀdwa: to be corrected wherever it occurs (unless it renders the written spelling). As a transcription of the Tigrinya place name, it is no doubt ʿAdwā.

**Comment by eu-genia** [comment]
the modern-day Tgn standard is ʿĀdwā https://ti.wikipedia.org/wiki/%E1%8B%93%E1%8B%B5%E1%8B%8B

**Comment by eu-genia** [comment]
(Amharic has first order alif, Tigrinya seems to have fourth order ayn as the first letter, at least in present-day usage)

**Comment by eu-genia** [comment]
![image](https://user-images.githubusercontent.com/17987728/183035468-b3a3b3d8-3a79-444c-9041-bd395b90111d.png)

**Comment by DenisNosnitsin1970** [comment]
Yes, this is true. As far as I know, this is how it is in the "post 1991" orthography reforms in Tigrigna (both Tigray and Eritrea). According to the new orthography, ዐድዋ would phonetically result in ʿÄdwa, therefore ዓድዋ is used. አድዋ  is Amharic (ኣድዋ seems to occur but is very rare).

**Comment by eu-genia** [comment]
I suggest that we try to follow the script; i.e. in present-day contexts (mentioning the actual town or region today) with stick with modern standard orthography (ዓድዋ , in administrative context also አድዋ ). 
When transcribing historical documents esp. those in Geez we use the spelling as given in the source (ዐድዋ ).
The entries
LOC1262Adwa
LOC1263Adwa
LOC1264Adwa
could have something like
```
               <placeName xml:lang="gez" xml:id="n1">ዐድዋ፡</placeName>
               <placeName xml:lang="gez" corresp="#n1" type="normalized">ʿAdwā</placeName>
               <placeName xml:lang="ti" xml:id="n2">ዓድዋ፡</placeName>
               <placeName xml:lang="ti" corresp="#n2" type="normalized">ʿĀdwā</placeName>
               <placeName xml:lang="am" xml:id="n3">አድዋ፡</placeName>
               <placeName xml:lang="am" corresp="#n3" type="normalized">ʾAdwā</placeName>
               <placeName xml:lang="am" xml:id="n4" type="alt">ዓዱዋ፡</placeName>
``` etc.
